### DISCUSSION TITLE
Include Web View in another node?

### DISCUSSION THREAD

**Discussion by XaverStiensmeier** [TriliumNext/Trilium#4627] [Q&A] [answered]
Is it possible to include a web view in another node such that it renders the page as an image or something similar? I know about just dragging it their in order to get a link - which is not bad - but for my use case an image/full view would be best.

**Comment by meichthys** [comment]
Ironically enough you are looking for the aptly named "include" feature. 
![Screenshot_20240131-121954.png](https://github.com/zadam/trilium/assets/10717998/f5b31bae-f9be-4d6c-b41f-50769d21dc04)

**Reply by XaverStiensmeier** [reply]
But this doesn't work with web views - or at least not on default:
![image](https://github.com/zadam/trilium/assets/36056823/d986a3c0-5682-4453-8edf-4d9828ac240b)

It just shows the icon symbol.

**Reply by meichthys** [reply]
I see what you mean now. I hadn't tried this with the Include feature.

**Reply by zadam** [reply, marked as answer]
Not all note types are supported for the "include note". Maybe webview could be supported.

Note that webview is electron feature, and its future is not certain. It might be removed in a future release.

**Reply by XaverStiensmeier** [reply]
I see. I don't know much about electron, but it is good to know that it might be dropped in the future.
I am going to open a feature request for webview include support.
